registry-requirements.md

```diff
@@ -122,3 +69,48 @@ Same content as existing binding
 (Think about how to manage the set of semantic artifacts/ontologies)
 
 Create a repository template to make it easy for people to create new bindings
+
+## Entry Requirements
+
+
+## Basic Requirements
```

Comment:
Appears to be meant to be a subsection of `Entry Requirements`
```suggestion
### Basic Requirements
```

Reply:
Not in this case. The entry requirements will be what is written in the registry section. The basic requirements are a big set of requirements and will be moved into entry requirements in some cases. There is still some work to do in this PR so I am putting it into the draft status.